src/libraries/System.Runtime.Loader/tests/AssemblyExtensionsTest.cs

```diff
@@ -12,9 +12,7 @@ class NonRuntimeAssembly : Assembly
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/45689", platforms: TestPlatforms.AnyUnix, runtimes: TestRuntimes.Mono)]
-        [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnMono("Not yet implemented on Mono")]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/45689", platforms: ~TestPlatforms.Windows, runtimes: TestRuntimes.CoreCLR)]
```

Comment:
I found out the hard way but ActiveIssue doesn't actually affect whether the test runs.  The test is going to fail on Linux/MacOS on coreclr

Reply:
I'm not totally convinced.  Just tried
```
        [Fact]
        [ActiveIssue("https://github.com/dotnet/runtime/issues/45689", platforms: ~TestPlatforms.Windows, runtimes: TestRuntimes.Mono)]
        public static void FailingTest()
        {
                Assert.True(false);
        }
```

and it skipped the test on Linux using Mono.  and when I changed to `TestPlatforms.CoreCLR` running on Mono on Linux ran the test and failed.

Reply:
> I found out the hard way but ActiveIssue doesn't actually affect whether the test runs. 

It should.  If it doesn't, that's a bug somewhere.

Reply:
runtime (Libraries Test Run release coreclr OSX x64 Debug) passed, so looks like it's working.

Reply:
I must have had things backwards or something.